File: app/src/main/java/vinova/henry/com/hotfilm/feature/detail/DetailPresenterImp.java
package vinova.henry.com.hotfilm.feature.detail;

import java.util.ArrayList;
import java.util.List;

import vinova.henry.com.hotfilm.models.Trailer;

public class DetailPresenterImp implements IDetailContract.IDetailPresenter{
    @Override
    public String[] createSpinnerItem(List<Trailer> trailers) {

        String[] a = new String[trailers.size()];
        for (int i = 0; i < trailers.size(); i++){
            a[i] = trailers.get(i).getName();
        }
        return a;
    }
}

File: app/src/main/java/vinova/henry/com/hotfilm/feature/detail/DetailPresenterImp.kt
package vinova.henry.com.hotfilm.feature.detail

import java.util.ArrayList

import vinova.henry.com.hotfilm.models.Trailer

class DetailPresenterImp : IDetailContract.IDetailPresenter {
    override fun createSpinnerItem(trailers: List<Trailer>): Array<String> {

        var a = Array<String>(trailers.size){""}
        for (i in 0..trailers.size) {
            a[i] = trailers.get(i).name.toString()
        }
        return a
    }
}
